hmpps-devops-tools/Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN curl -sLo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-r
 
 RUN curl -L https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz | tar xz && mv linux-amd64/helm /bin/helm && rm -rf linux-amd64
 
+RUN curl -sL https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | tee /etc/apt/trusted.gpg.d/postgresql.asc.gpg > /dev/null \
```

Comment:
have to grab v14 as some of our servers are postgres v14 and v14 isn't available by default in debian bullseye

Reply:
Do we need to make this configurable as many different versions of Postgres are in use in various projects? Or are we ok with the latest version?